main/ertp/ERTP-Guide/Overview.md

```diff
@@ -0,0 +1,260 @@
+# ERTP Guide
+
+[ERTP](https://agoric.com/documentation/glossary/#ertp) (*Electronic
+Rights Transfer Protocol*)
+is Agoric's token standard for transferring tokens and other digital assets in
+JavaScript. Using the
+[ERTP API](https://agoric.com/documentation/ertp/api/),
+you can easily create and use digital assets, all of which are
+transferred exactly the same way and with exactly the same security properties. 
+
+ERTP uses
+*[object capabilities](https://agoric.com/documentation/glossary/#object-capabilities)*
+to enforce access control. If your program has a reference to an
+object, it can call methods on that object. If it doesn't have a
+reference, it can't. For more on object capabilities, see
+[Chip Morningstar's post](http://habitatchronicles.com/2017/05/what-are-capabilities/).
+
+## Objects Overview
+
+![ERTP object relationships](./assets/relationships.svg)  
+
+There are eight fundamental ERTP objects, two of which are parts of
```

Comment:
Mentioning the exact number and which are parts of another does not seem to be helpful for the reader. The reader wants to know 1) what ERTP is and 2) how they can use it quickly. This is not either of those.

Reply:
Also, I think we mention objects a lot. Why? Why say `payment object` when we could just say `payment`?

Reply:
Honestly, a fair number of those were to be able to put the "s" on "objects" rather than `purse`s, which looks odd.

Reply:
ah, ok! I think `purses` looks fine if we need to make it plural.